tools/run_tests/performance/build_performance.sh

```diff
@@ -43,6 +43,9 @@ do
   "go")
     tools/run_tests/performance/build_performance_go.sh
     ;;
+  "php")
```

Comment:
now you can add a special case for php_protobuf_c  here  and revert the changes  around `_with_argv` you've added.

Reply:
Done.